docs/sofar.rst

```diff
@@ -0,0 +1,36 @@
+sofar
+=====
+
+.. toctree::
+   :caption: Getting Started
+   :maxdepth: 1
+
+   readme
+
+
+.. toctree::
+   :caption: Documentation
+   :maxdepth: 2
+
+   modules
```

Comment:
```suggestion
   :maxdepth: 2

   modules/sofar
```
then it would be
sofar
- SOFA
- equals
- all other functions
to also show the subfunctions of the class use ``:maxdepth: 3``